Cornelica/scenario/013_03a.txt

```diff
@@ -90,7 +90,7 @@
 @bs f=ユスティーナ@笑み body=-1 face=0 x=350 y=50 t=100 wt=0
 @bs f=アル@困り body=-1 face=0 x=-350 y=50 t=200
 
-[Justina] "So you're Al, are you shopping too? Now if are, I'll have to add an additional charge of [extra] ... ♪" ♪
+[Justina]"So you're Al, are you shopping too? \nNow you if are, I'll have to add \nan additional charge of extra♪"
```

Comment:
maybe `Now if you are,` - lol machine translate fail

Reply:
Yep... been italian don't help with this typo